src/java.base/linux/native/libnio/ch/FileDispatcherImpl.c

```diff
@@ -91,7 +91,7 @@ Java_sun_nio_ch_FileDispatcherImpl_transferTo0(JNIEnv *env, jobject this,
     if (append == JNI_TRUE)
         return IOS_UNSUPPORTED_CASE;
 
-    off64_t offset = (off64_t)position;
+    loff_t offset = (loff_t)position;
```

Comment:
Is this `loff_t` for the benefit of `copy_file_range`?

Reply:
That is how copy_file_range is defined. The cast to off64_t was technically incorrect (but they ended up being the same type).